gclient/src/api/storage/mod.rs

```diff
@@ -58,6 +58,13 @@ impl GearApi {
         Ok(data.map(|(m, i)| (m.into(), i)))
     }
 
+    /// Get up to `count` messages from the mailbox.
+    pub async fn get_all_mailbox(&self, count: u32) -> Result<Vec<(StoredMessage, Interval<u32>)>> {
+        let data: Vec<(stored::StoredMessage, Interval<u32>)> =
```

Comment:
No need to be so explicit.
```suggestion
        let data =
```